### PR TITLE
[TW-1203] Flows visible on API network

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Added a single line to the overview mentioning flows since they appear in the sidebar.

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/public-api-network/public-api-network-overview.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ updated: 2023-03-30
...
+Postman has always been passionate about successful API calls and supporting greater collaboration and innovation. Postman Collections and workspaces provide developers with the tools they need for successful API calls. [Postman Flows](/docs/postman-flows/gs/flows-overview/) enable you to build and collaborate on API-driven applications. [Public workspaces](/docs/collaborating-in-postman/using-workspaces/public-workspaces/) enable API developers to [view, share, and consume your APIs](https://www.postman.com/api-platform/api-catalog/) and all the elements you publish, and it’s free of charge. It has never been easier to present your APIs to the global API community. With more than 57,000 public workspaces and 125,000 collections already in the network, it has become the world’s largest public API network.
```
Comment (nerminamiller-postman) [inline src/pages/docs/collaborating-in-postman/public-api-network/public-api-network-overview.md:8]:
```suggestion
Postman gives you the tools you need to make successful API calls. Use [Postman Flows](/docs/postman-flows/gs/flows-overview/) to build and collaborate on API-driven applications. Leverage [Public workspaces](/docs/collaborating-in-postman/using-workspaces/public-workspaces/) to [view, share, and consume your APIs](https://www.postman.com/api-platform/api-catalog/) and all the elements you publish.
```

Comment (nerminamiller-postman) [reply src/pages/docs/collaborating-in-postman/public-api-network/public-api-network-overview.md]: @petewitcher-postman feel free to accept or reject any part of my suggestion.

**Review by nerminamiller-postman — APPROVED** [review]
@petewitcher-postman, left you a small suggestion

File: src/pages/docs/collaborating-in-postman/public-api-network/public-api-network-overview.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (petewitcher-postman) [reply src/pages/docs/collaborating-in-postman/public-api-network/public-api-network-overview.md]: I added flows where appropriate, but this page has a lot of drift and some of the bullets aren't valid. Created [TW-1217](https://postmanlabs.atlassian.net/browse/TW-1217?atlOrigin=eyJpIjoiNzM1NmQ5MDI5ZTNkNDZjNDg5YWNkYTI0M2Q0NmUzZTkiLCJwIjoiaiJ9) to address it.

**Pull request merged**